organisaatio-ui/src/components/Sivut/Ryhmat/Muokkaus/MuokkausLomake.tsx

```diff
@@ -0,0 +1,207 @@
+        (rt) =>
+            ryhma.kayttoryhmat.map((k: string) => dropKoodiVersionSuffix(k)).includes(rt.value) ||
+            ryhma.kayttoryhmat.includes(rt.value)
+    );
+    const ryhmaTyypit = ryhmaTyypitOptions.filter(
+        (rt) =>
+            ryhma.ryhmatyypit.map((k: string) => dropKoodiVersionSuffix(k)).includes(rt.value) ||
+            ryhma.ryhmatyypit.includes(rt.value)
+    );
+
+    return (
+        <PohjaSivu>
+            <div className={styles.YlaBanneri}>
+                <div>
+                    <a href="/organisaatio/ryhmat">
+                        <Icon icon={homeIcon} />
+                    </a>
+                </div>
+                <div>
+                    <a href="/organisaatio/ryhmat">{i18n.translate('KAIKKI_RYHMAT')}</a>
+                </div>
+            </div>
+            <div className={styles.PaaKehys}>
+                <div className={styles.ValiContainer}>
+                    <div className={styles.ValiOtsikko}>
+                        <h3>{i18n.translate('RYHMA')}</h3>
+                        <h1>{mapLocalizedKoodiToLang(language, 'nimi', ryhma)}</h1>
+                    </div>
+                </div>
+                <div className={styles.PaaOsio}>
+                    <div className={styles.OtsikkoRivi}>
+                        <div className={styles.Otsikko}>
+                            <h3>{i18n.translate('RYHMAN_TIEDOT_OTSIKKO')}</h3>
+                        </div>
+                    </div>
+                    <div className={styles.OidRivi}>
+                        <span className={styles.AvainKevyestiBoldattu}>{i18n.translate('OID')}</span>
+                        <span className={styles.ReadOnly}>{ryhma.oid || ''}</span>
+                    </div>
+                    <div className={styles.Rivi}>
+                        <div className={styles.Kentta}>
+                            <label>{i18n.translate('RYHMAN_NIMI')}</label>
+                            <div className={styles.Rivi}>
+                                <label>{i18n.translate('SUOMEKSI')}</label>
+                                <div className={styles.PitkaInput}>
+                                    <Input {...nimiFiBind} />
+                                </div>
+                            </div>
+                            <div className={styles.Rivi}>
+                                <label>{i18n.translate('RUOTSIKSI')}</label>
+                                <div className={styles.PitkaInput}>
+                                    <Input {...nimiSvBind} />
+                                </div>
+                            </div>
+                            <div className={styles.Rivi}>
+                                <label>{i18n.translate('ENGLANNIKSI')}</label>
+                                <div className={styles.PitkaInput}>
+                                    <Input {...nimiEnBind} />
+                                </div>
+                            </div>
```

Comment:
i think its clearer this way and the file is not too many lines? do you want it changed?

Reply:
I dont _want_ anything. I just make notes about the code. If we imagine that message key would be incorporated in those *Bind -objects, then this would be easy to do with loop which have some benefits